### DISCUSSION TITLE
Make Private FIle shorter?

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#2290] [Q&A]
I'm doing in develop branch and I have tried Private File. 

What do you think if we make a temp url for Private Files?

For Ex: `/api/files/{temp_path}/{collectionId|/{recordId}/file_name.png`
Or only need `temp_path` because we can get **Collection** and **Record** ID from it :D 

The `temp_path` will be refreshed every 5 or 10 mins

Currently I'm feeling the URL is too long..

**Comment by ganigeorgiev** [comment]
I don't understand what is the end goal. Private files works with a with a unique request file token based on which we check the View API rule of the related collection. It is not just some random string that changes every 5 mins.

> Currently I'm feeling the URL is too long..

Why the file url length matter at all?
